lib/build.js

```diff
@@ -20,11 +21,12 @@ iterator(function (srcname, name, variant, meta) {
   var md = fs.createWriteStream(`${__dirname}/../docs/${name}/${name}.md`);
   var html = pandoc({
     "--metadata-file": `${__dirname}/../${srcname}/${variant}.yaml`,
+    ...meta.pandocOptions,
```

Comment:
Something like this is missing in `server.js`

Reply:
Clicking on an XML link still shows
~~~
Cannot GET /odata-vocabularies/Org.OData.Core.V1.xml
~~~

Reply:
Cannot reproduce: which link did you click?

Reply:
Now it works 🤷🏻‍♂️ 😄

Reply:
I cannot reproduce that, the selector
```
a[href]:not([href^=http]):not([href^=\#])
```
finds nothing on odata-vocabularies.html.

Which link did you click?

Reply:
Sorry, I cannot reproduce that, the CSS selector
```
a[href]:not([href^=http]):not([href^=\#])
```
finds nothing in odata-vocabularies.html.